gateway-service-definitions/src/main/resources/services/livy/0.4.0/rewrite.xml

```diff
@@ -16,33 +16,10 @@
    limitations under the License.
```

Comment:
Don't make these unrelated changes? You look to be trying to remove `/v1` from the url handling which isn't necessary.